lib/Controller/ApiController.php

```diff
@@ -1161,7 +1161,8 @@ public function insertSubmission(int $formId, array $answers, string $shareHash
 
 		// Ensure the form is unique if needed.
 		// If we can not submit anymore then the submission must be unique
-		if (!$this->formsService->canSubmit($form) && !$this->submissionService->isUniqueSubmission($submission)) {
+		if (!$this->formsService->canSubmit($form) && $this->submissionMapper->hasMultipleFormSubmissionsByUser($form, $submission->getUserId())
+		) {
```

Comment:
I'd like to keep the braces in the first line like everywhere else :)

```suggestion
		if (!$this->formsService->canSubmit($form) && $this->submissionMapper->hasMultipleFormSubmissionsByUser($form, $submission->getUserId())) {
```